podman_compose.py

```diff
@@ -833,6 +833,12 @@ def get_network_create_args(net_desc, proj_name, net_name):
     ipam_config_ls = ipam.get("config", [])
     if net_desc.get("enable_ipv6"):
         args.append("--ipv6")
+    if net_desc.get("ignore"):
+        args.append("--ignore")
```

Comment:
Could you tell the use case for this? It should be easy enough to set the network to external, so there must be something I don't understand.

Reply:
When you run `podman-compose up` it will create the pod, the containers and the network. If you run `podman-compose down` it will destroy the pod, the containers, but not the network.

By allowing the use of `--ignore` one can ensure that multiple up/down will work and not fail because network already exists.

Yes, it is possible (and easy) to use the external network, but this means that the first `up` is different than another one.

It's most useful while experimenting with service composes rather than in production (which is my primary use for `podman-compose`).

Reply:
@p12tic I removed the "--ignore" setting from the patch. In the end, using "name: <network>" is enough to re-use an existing network (at least in the case I had).